test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Wire.fir

```diff
@@ -41,9 +42,23 @@ circuit Top :
     out.multivec[1][2] <= w.multivec[1][2]
     out.uint <= w.uint
 
+  extmodule Tap:
+    output clock: Clock
+    output a: UInt<1>
+    input b: UInt<1>
+
+
+  ; MYVIEW_COMPANION: define RANDOM
```

Comment:
Can you add a note here about what this is checking? Our future selves may be very confused by this... 😅

Reply:
That's right :)